docs/src/sw.js

```diff
@@ -145,9 +145,13 @@ self.addEventListener('fetch', event => {
         return responseNetwork
       })
       .catch(() => {
-        // User is landing on our page.
         if (event.request.mode === 'navigate') {
-          return global.caches.match('./')
+          // If user navigation failed, let's provide an offline fallback
+          // to inform them properly.
+          //
+          // offline-fallback.html should be defined in servierWorkerOption.assets
+          // to ensure it's always cached.
+          return global.caches.match('./offline-fallback.html')
```

Comment:
Yes, this default makes a lot more sense to me. In order for it to work though, the offline-fallback.html page needs to be added to the cache first. Could you update line 15 accordingly.

I don't agree that serviceWorkerOptions.assets is, necessarily the right place to make sure the fallback page is cached. Perhaps define a constant in this file that defines the fallback page URL. For documentation purposes, something like this might work:

```js
// Page to fallback to in case the resource we are trying to navigate to can't be found on
// either cache or network. Hint: Use `transformOptions` to add additional information to
// serviceWorkerOption
const FALLBACK_PAGE = serviceWorkerOption.fallbackPage || './offline-fallback.html';
```